src/python/pants/core/goals/test.py

```diff
@@ -228,6 +236,12 @@ def register_options(cls, register) -> None:
             default=False,
             help="Force the tests to run, even if they could be satisfied from cache.",
         )
+        register(
+            "--show-output",
```

Comment:
How about `--output`? I expect this to be used on the CLI frequently, so short is good. `./pants test --output=all`.